dracut/40clevis-unlock/clevis-unlock.service

```
@@ -0,0 +1,23 @@
+[Unit]
+Description=Unlock disks using Clevis
+DefaultDependencies=no
+# clevis-luks-generic-unlocker depends on executables in /sysusr/usr.
+RequiresMountsFor=/sysusr/usr/
+# dracut-initqueue will wait for /dev/disk/by-label/ROOT, so we must decrypt the root disk before dracut-initqueue.
+Before=dracut-initqueue.service
+Wants=systemd-networkd.service
+After=systemd-networkd.service
+Wants=systemd-resolved.service
+After=systemd-resolved.service
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+# We let the unlocker run in "loop" mode: it will keep retrying unlocking until there are no more devices to unlock.
+# This is important if network-bound disk encryption with Tang is used. Even after network-online.target is reached
+# it can happen that the network is not ready yet and the request to the Tang server fails.
+Type=simple
```

```suggestion
Type=oneshot
RemainAfterExit=yes
```
If we want to mount the rootfs we need to make sure we can use this service as dependency for it.

---

Scratch that, I see that looping mode is used.